static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -569,7 +569,8 @@ function FlamegraphZoomView({
       // rendered on the flamegraph are removed from the flamegraphView
       setConfigSpaceCursor(null);
 
-      if (evt.metaKey) {
+      // pinch to zoom is recognized as `ctrlKey + wheelEvent`
+      if (evt.metaKey || evt.ctrlKey) {
```

Comment:
Could we have some platform distinction here, else we are just duplicating shortcuts here?

Reply:
For future reference if anyone comes back ...
https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent
![image](https://user-images.githubusercontent.com/7349258/175096021-76badf23-1f0b-433d-b085-7e5fa4b66f88.png)